app/views/publications/v2/monthly_statistics/show.html.erb

```diff
@@ -0,0 +1,309 @@
+          <span class="app-toc__number">4.</span>
+          <span class="app-toc__content">Candidate sex</span>
+        </a>
+      </li>
+      <li>
+        <a href="#applications-by-area" class="govuk-link app-toc__link">
+          <span class="app-toc__number">5.</span>
+          <span class="app-toc__content">Candidate area</span>
+        </a>
+      </li>
+      <li>
+        <a href="#applications-by-course-phase" class="govuk-link app-toc__link">
+          <span class="app-toc__number">6.</span>
+          <span class="app-toc__content">Course phase</span>
+        </a>
+      </li>
+      <li>
+        <a href="#applications-by-route" class="govuk-link app-toc__link">
+          <span class="app-toc__number">7.</span>
+          <span class="app-toc__content">Route into teaching</span>
+        </a>
+      </li>
+      <li>
+        <a href="#by-primary-specialist-subject" class="govuk-link app-toc__link">
+          <span class="app-toc__number">8.</span>
+          <span class="app-toc__content">Primary specialist subject</span>
+        </a>
+      </li>
+      <li>
+        <a href="#by-secondary-subject" class="govuk-link app-toc__link">
+          <span class="app-toc__number">9.</span>
+          <span class="app-toc__content">Secondary subject</span>
+        </a>
+      </li>
+      <li>
+        <a href="#by-provider-region" class="govuk-link app-toc__link">
+          <span class="app-toc__number">10.</span>
+          <span class="app-toc__content">Provider region</span>
+        </a>
+      </li>
+      <li>
+        <a href="#downloads" class="govuk-link app-toc__link">
+          <span class="app-toc__number">11.</span>
+          <span class="app-toc__content">Download the data</span>
+        </a>
+      </li>
+      <li>
+        <a href="#footnotes" class="govuk-link app-toc__link">
+          <span class="app-toc__number">12.</span>
+          <span class="app-toc__content">Footnotes</span>
+        </a>
+      </li>
+    </ol>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l" id="introduction">1. Introduction</h2>
+
+    <p class="govuk-body">These statistics cover applications for courses in England starting in the <%= @presenter.academic_year_name %> academic year. To allow for comparison statistics covering the <%= @presenter.current_cycle_name %> academic year are also included.</p>
+
+    <p class="govuk-body">New definitions and methodology have been introduced for the <%= @presenter.academic_year_name %> academic year because candidates can now submit applications to courses individually up to a maximum limit of 4 open applications at any one time. Previously candidates submitted an application form containing up to 4 applications at the same time, all of which had to be processed before the candidate could submit another application form that cycle.</p>
```

Comment:
not a blocker – but thoughts on using the `MAXIMUM_NUMBER_OF_COURSE_CHOICES` constant?

Reply:
I think that's a good idea. I can do that now